src/ol_concourse/pipelines/container_images/jupyter_courses.py

```diff
@@ -44,6 +46,41 @@ class CourseImageInfo:
         repo_uri="git@github.mit.edu:ol-notebooks-qa/course_clustering_and_descriptive_ai.git",
         image_name="clustering_and_descriptive_ai",
     ),
+    CourseImageInfo(
+        course_name="UAI_SOURCE-UAI.6-3T2025",
+        repo_uri="git@github.mit.edu:ol-notebooks-qa/UAI_SOURCE-UAI.6-3T2025.git",
+        image_name="UAI_SOURCE-UAI.6-3T2025",
```

Comment:
```suggestion
        image_name="uai_source-uai.6",
```
Including a term specifier seems like it will become problematic. If we need to include that information it should probably be set as a label on the image.

Also, only lowercase letters are permitted in the image name https://docs.aws.amazon.com/AmazonECR/latest/userguide/repository-create.html

Reply:
No issue taking the term specifier out of the image name and/or the course name (nothing upstream or downstream really cares, and it's only in the repo name because it matches the mitxonline GH repo naming scheme) - I'll it pulled out of both for the time being. It's easy enough to change that if we need the info later.

Ah, didn't realize about those naming restrictions; will get that fixed shortly!